dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/scheduling/SchedulingManagerTest.java

```diff
@@ -109,6 +109,9 @@ void setUp() {
     when(cacheProvider.createRunningJobsInfoCache()).thenReturn(new TestCache<>());
     when(cacheProvider.createCompletedJobsInfoCache()).thenReturn(new TestCache<>());
 
+    LeaderManager leaderManager = mock(LeaderManager.class);
+    when(leaderManager.isLeader()).thenReturn(true);
```

Comment:
FYI: This illustrates quite well the change of user created job now being "leader only" as well. So to correct the test setup for this we now need to pose as leader to have them executed.